.changelog/4964.cfg.md

```diff
@@ -0,0 +1,8 @@
+go/consensus/tendermint: Refactor seed node
+
+Until now a seed node was a Tendermint backend-specific construct which
+supported only one service. To provide support for other services we had
+to refactor the seed node and move Tendermint seed construct to a service.
+Configuration of the latter stayed the same. However, to run a node as
+a seed node one has to specify `--mode seed` flag or add `mode: seed`
+setting to the config file.
```

Comment:
After this change also the various addresses need to be changed? E.g. to use public keys instead of those tendermint addresses?